java/ql/src/change-notes/2025-07-17-spring-actuators-config-promo.md

```diff
@@ -0,0 +1,4 @@
+---
+category: newQuery
+---
+* The query `java/insecure-spring-actuator-config` has been promoted from experimental to the main query pack as `java/spring-boot-exposed-actuators-config`. Its results will now appear by default. This query was originally submitted as an experimental query [by @luchua-bc](https://github.com/github/codeql/pull/5384).
```

Comment:
Could you add a sentence summarising what the query does?

Reply:
Done in https://github.com/github/codeql/pull/20006/commits/f0542dd828850020d8f673104bfedd436f46b27a